### ISSUE TITLE
Buttons need to be arranged more neatly

### ISSUE THREAD

**Issue by theharrisonmoore** [yalla-coop/earwig#977]
Buttons need to be arranged more neatly

![IMG_0449](https://user-images.githubusercontent.com/45825100/72153302-7728bb80-33a5-11ea-9c15-9ba753c2e9c7.jpg)

**Comment by thejoefriel** [comment]
Hey @theharrisonmoore sorry not quite sure how you want this to look from the photo above? Do you have a wireframe you can share please?

**Comment by theharrisonmoore** [comment]
Hi Joe, I just mean the report icon should be on the same level
(vertically) as the other icons. At the moment, it's sitting slightly lower
if you look closely. It's really noticeable on mobile.

Best,
Harrison Moore
Sparky & Founder
*earwig <http://earwigwork.com>*

ᐧ

**Comment by thejoefriel** [comment]
Hey @theharrisonmoore I think you're referring to a different issue. This one is about the buttons on the thank you page :)

**Comment by theharrisonmoore** [comment]
Yes, I may be. Having spoken to Simon about not having Facebook Messenger
on mobile, I guess my overall comment now is to ensure that the buttons,
whether on Thank You page or Build Your Network page, are arranged neatly
and without a gap where the missing button should be.

Best,
Harrison Moore
Sparky & Founder
*earwig <http://earwigwork.com>*

ᐧ
